anvil/anvil.go

```diff
@@ -272,6 +262,10 @@ func (a *Anvil) Web3ClientVersion(ctx context.Context) (string, error) {
 }
 
 // eth_ API
+func (a *Anvil) EthGetCode(ctx context.Context, account common.Address) ([]byte, error) {
```

Comment:
not imporant in this PR. I think externally we should just use `ethclient` externally instead of creating these stubs here

Reply:
yeah that makes sense!